### PR TITLE
Reverts Pillcreep

### PR DESCRIPTION
**Pull request by jknpj**
TODO: unspaghetti robot upgrades

:cl:
 * rscdel: The medical cyborg mk-2 upgrade doesn't upgrade the chemistry gripper anymore.
 * rscdel: Removed the anomaly research requirement for the medical cyborg mk-2 upgrade.

### CODE REVIEW

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -250,7 +207,23 @@
 	R.illegal_weapons = TRUE
 	R.SetEmagged()
 
-/obj/item/borg/upgrade/engineering/
+//Medical Stuff
+/obj/item/borg/upgrade/medical
+	name = "medical cyborg MK-2 upgrade board"
+	desc = "Used to give a medical cyborg advanced care tools and upgrade their chemistry gripper to be able to handle pills and pill bottles."
```
Comment (Kurfursten) [inline code/game/objects/items/robot/robot_upgrades.dm]: desc

**Comment by SonixApache** [comment]
seperate gripper where

File: code/game/objects/items/robot/robot_upgrades.dm (hunk 1 above)
Comment (jknpj) [reply code/game/objects/items/robot/robot_upgrades.dm]: didn't i fucking commit the desc update

**Comment by Hoshi-chan** [comment]
Holy shit, it's about time

**Pull request merged**